config/locales/telephony/zh.yml

```diff
@@ -18,7 +18,7 @@ zh:
         %{app_name}: 你的一次性代码是 %{code}。此代码在 %{expiration} 分钟后作废。请勿与任何人分享此代码。
 
         @%{domain} #%{code}
-      voice: 你好! 你的 %{format_length}-%{format_type} %{app_name} 一次性代码是 %{code}。你的一次性代码是 ，%{code}。重复一下，你的一次性代码是 %{code}。此代码 %{expiration} 分钟后会作废。
+      voice: 你好! 你的%{format_length}%{format_type}一次性代码是 %{code}。你的一次性代码是 %{code}。 重复一下，你的一次性代码是 %{code}。此代码 %{expiration} 分钟后会作废。
```

Comment:
Re: https://github.com/18F/identity-idp/pull/10757#issuecomment-2148064301, is there a ticket to follow-up to re-add "Login.gov" for the Chinese translations?

Reply:
I will post about a follow-up ticket in the acceptance thread for this